frontend/src/components/attribute/attribute_group_list.vue

```diff
@@ -440,7 +440,7 @@ export default Vue.extend({
         }
         let attribute_value = attr_values[attribute_group_id]
         let attribute_group = this.attribute_group_list.find(group => group.id === attribute_group_id)
-        if(!attribute_value){
+        if(!attribute_value || !attribute_group){
```

Comment:
This fixes a runtime exception that only happens for image file annotations that are not part of the compound file

```vue.esm.js:628 [Vue warn]: Error in render: "TypeError: Cannot read properties of undefined (reading 'kind')"

found in

---> <AttributeGroupList> at src/components/attribute/attribute_group_list.vue
       <VExpansionPanelContent>
         <VExpansionPanel>
           <VExpansionPanels>
             <GlobalAttributesList> at src/components/attribute/global_attributes_list.vue
               <InstanceDetailList> at src/components/annotation/image_and_video_annotation/instance_detail_list_view.vue
                 <VNavigationDrawer>
                   <SidebarImageAnnotation> at src/components/annotation/image_and_video_annotation/sidebar_image_annotation.vue
                     <SidebarFactory> at src/components/annotation/sidebar_factory.vue
                       <AnnotationUiFactory> at src/components/annotation/annotation_ui_factory.vue
                         <VMain>
                           <VApp>
                             <App> at src/App.vue
                               <Root>
                               ```